server/src/main/java/org/opensearch/extensions/ExtensionsSettings.java

```diff
@@ -42,12 +42,7 @@ public static class Extension {
         private String hostAddress;
```

Comment:
I commented on [SDK #65](https://github.com/opensearch-project/opensearch-sdk-java/issues/65) that I don't see a need for both `hostName` and `hostAddress` because one should resolve to the other (or you can put IP in for name). Would like to see that updated here.

Reply:
Sure, we have inherited the `hostname` param from `DiscoveryNode`.
I believe the usage of `hostname` is used to identify between different hosts but for extensions (for now) it doesn't matter. I'll replace the hostName with address.